Eatery Blue/UI/EateryScreen/EateryModelController.swift

```diff
@@ -19,6 +19,7 @@ class EateryModelController: EateryViewController {
         return formatter
     }()
 
+    var menuHasLoaded = false
```

Comment:
```suggestion
    var menuHasLoaded: Bool = false
```
Just to keep it consistent.